services/PostService.js

```diff
@@ -0,0 +1,19 @@
+const mongoose      = require('../mongoose');
+const Post          = require('../models/Post');
+
+class PostService {
+    getOnePost(post_id) {
+        return Post.find({_id: post_id});
+    };
+
+    getAllPosts() {
+        return Post.find({});
+    };
+
+    postNewPost(title, author, body) {
```

Comment:
Why is this function called `postNewPost`? The HTTP method POST is defined in the HTTP protocol, but is not relevant to your `PostService`. Something more natural like `createNewPost` should probably be used

Reply:
fixed